spec/beaker/host/unix_spec.rb

```diff
@@ -37,13 +37,5 @@ module Unix
         expect(host.determine_ssh_server).to be === :openssh
       end
     end
-
-    describe '#validate_setup' do
```

Comment:
Should we drop `validate_setup` altogether, since it's not used anymore? We're doing breaking changes anyway.

Reply:
sounds good to me. https://github.com/voxpupuli/beaker/pull/1875